tests/MartinGeorgiev/Doctrine/Fixtures/Entity/ContainsDates.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\MartinGeorgiev\Doctrine\Fixtures\Entity;
+
+/**
+ * @Entity
+ */
+class ContainsDates extends Entity
+{
+    /**
+     * @var \DateTime
+     *
+     * @Column(type="date")
```

Comment:
Why `$date1` and `$date2` rely on different `\DateTimeInterface` implementations? Wdyt about using the same type, preferably an immutable one?

Reply:
In the future tests we might need as mutable date as immutable version.

Reply:
Wdyt about waiting for such a need to emerge? Imo let's update with tests for both if (once) we cross that bridge.

Reply:
Lovely - thank you very much :)